metecho/api/serializers.py

```diff
@@ -88,6 +89,80 @@ def get_choices(self, cutoff=None):  # pragma: nocover
         return OrderedDict(((item.pk, self.display_value(item)) for item in queryset))
 
 
+class GitHubIssueSerializer(HashIdModelSerializer):
+    project = serializers.PrimaryKeyRelatedField(
+        queryset=Project.objects.all(), pk_field=serializers.CharField()
+    )
+    epic = serializers.SerializerMethodField()
+    task = serializers.SerializerMethodField()
+
+    class Meta:
+        model = GitHubIssue
+        read_only_fields = (
+            "id",
+            "number",
+            "title",
+            "created_at",
+            "html_url",
+            "project",
+            "epic",
+            "task",
+        )
+        fields = read_only_fields
+
+    @extend_schema_field(
+        {
+            "properties": {
+                "id": {"type": "string", "format": "HashID"},
+                "name": {"type": "string"},
+                "status": {"type": "string"},
+                "slug": {"type": "string"},
+            },
+            "nullable": True,
+        }
+    )
+    def get_epic(self, issue):
+        try:
+            return {
+                "id": str(issue.epic.id),
+                "name": issue.epic.name,
+                "status": issue.epic.status,
+                "slug": issue.epic.slug,
+            }
+        except Epic.DoesNotExist:
+            return None
+
+    @extend_schema_field(
+        {
+            "properties": {
+                "id": {"type": "string", "format": "HashID"},
+                "name": {"type": "string"},
+                "status": {"type": "string"},
+                "review_status": {"type": "string"},
+                "review_valid": {"type": "boolean"},
+                "pr_is_open": {"type": "boolean"},
+                "slug": {"type": "string"},
+                "epic_slug": {"type": "string", "nullable": True},
+            },
+            "nullable": True,
+        }
+    )
+    def get_task(self, issue):
+        try:
+            return {
+                "id": str(issue.task.id),
+                "name": issue.task.name,
+                "status": issue.task.status,
+                "review_status": issue.task.review_status,
+                "review_valid": issue.task.review_valid,
+                "pr_is_open": issue.task.pr_is_open,
+                "slug": issue.task.slug,
+                "epic_slug": issue.task.epic.slug if issue.task.epic else None,
```

Comment:
@jerivas This was breaking for tasks that don't have an epic, so I added this `if`.